src/store/store.ts

```diff
@@ -196,6 +197,10 @@ export const useStore = defineStore('store', {
       return this.stateModel.entityType
     },
 
+    getContinuationInBusinessInfo (): any {
```

Comment:
I'm guessing we might be able to create an interface for this later on when all the requirements are done right Sev?

Reply:
Exactly. I left a comment in continuation-in-state-interface.ts about this.